src/comments/comments.service.ts

```diff
@@ -61,7 +76,8 @@ export class CommentsService {
                 updated: new Date()
             }
         })
-        return updatedComment;
+        const action = await this.actionsService.addAction('Update',userId,'Comment',comment.id,updatedComment)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them